cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraCluster.java

```diff
@@ -448,6 +451,18 @@ private CassandraNode buildCassandraNode(@NotNull final Protos.Offer offer, fina
         }
     }
 
+    @NotNull
+    private RackDc getRackDc(@NotNull final Protos.Offer offer) {
+        final String rack = attributeValue(offer, RACK_ATTRIBUTE);
```

Comment:
Sure. Added comment with description.

Yes, the behavior is exactly the same if those attributes are not defined in offer (slave node).